### ISSUE TITLE
option to move all-day events

### ISSUE THREAD

**Issue by ilf** [lfos/calcurse#333]
Editing events with date/time gives the option "(5) Move".
Editing all-day events directly edits the description.
How can I edit the date of an all-day event?

**Comment by lhca** [comment]
You cannot, but see the discussion in #328.

**Comment by ilf** [comment]
Yeah, sorry for closing my feature request with a question. I noticed it's not possible, but I would think this is a useful (and logical) feature for many users.

I see #328, but it wants far more than this one. I am not sure I would want/need everything in #328, but I'll leave this one here open.

Thanks for all the work!
